incubator/quarkus/templates/default/pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>dev.appsody</groupId>
         <artifactId>quarkus</artifactId>
-        <version>[0.2,0.3)</version>
+        <version>[0.4,0.5)</version>
```

Comment:
`[0.3,0.4)`? See other comment

Reply:
I think I've confused things by doing two version updates in the same PR. I've managed to drag this out so long that Quarkus moved quicker. I'll simplify the PR to make it clearer what's changing.

Reply:
I just realised the `pom.xml` needs to be 0.3.0 too. Missed that in my first review sorry.

Reply:
@Kamran64 I've now fixed the commits to make a more logical set of changes. I also removed the double version bump as that's not needed anymore. Does it look right to you now?